prov/verbs/src/fi_verbs.c

```diff
@@ -970,7 +1006,10 @@ static int fi_ibv_getinfo(uint32_t version, const char *node, const char *servic
 		goto err2;
 
 	if (hints) {
-		ret = fi_ibv_check_hints(hints, verbs_info);
+		/* TODO if we do not bind to a specific device we have to check 
+		 * check_info list and return those fi_info structures that match hints
+		 */
```

Comment:
Can you adjust the wording on this comment?  It's not clear what needs to be done.  I think this comment may be addressing my concerns expressed in another response regarding fi_info not associated with a device, but I'm not sure.